src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let price = 0;
+
+  if (days < 3) {
+    price = days * 40;
```

Comment:
What do 40, 20 and 50 mean? Don't use mystery numbers. Move them to variables. BTW you use this value `days * 40` not once. Create a variable for it as well